GoInfoGame/GoInfoGameTests/quests/StepsRampTests.swift

```diff
@@ -40,15 +40,20 @@ final class StepsRampTests: XCTestCase {
      */
     
     func testStepsRampQuery() throws {
+        let fourYearsAgo = TestQuestUtils.olderThan(years: -4)
```

Comment:
both are same.. `fourYearsAgo` and `eightYearsAgo`. Why are two variables needed?

Reply:
removed redundant variable